tools/wake-format/README.md

```diff
@@ -0,0 +1,481 @@
+# wake-format
+
+`wake-format` is a tool for automatically applying the wake style guide to wake source files. It is intentionally opinionated to create a singular correct formatting standard.
+
+# Usage
+
+The standard usage of `wake-format` is to format a set of wake files in place.
+
+`wake-format -i source.wake source2.wake source3.wake`
+
+`wake-format` supports other flows for various use cases. The `-n`/`--dry-run` flag is ideal for CI/CD to check if the source needs to be formatted. It exits with a non-zero status code when a file would be changed by `wake-format`
+
+Running `wake-format` without `-i` will emit the file to `stdout` instead of overwritting it.
+
+# Documentation
+
+The information below should be helpful to contribute to `wake-format`
+
+## Building
+
+`wake-format` is built in the same manner as `wake`. Running `make` or `./bin/wake build default` from within the wake dir will produce the `wake-format` binary.
+
+The locally build `wake-format` can be found at `./bin/wake-format.native-cpp14-release`
```

Comment:
We should probably have the build system produce a normalized name for it, either via the makefile, or via wake's `installAs`

Reply:
Yeah we just add it to the installs with shim-wake, job-cache, and wakebox

Reply:
If you run `make install` then it _does_ get installed/renamed appropriately

```
> DESTDIR=../install make install
WAKE_PATH=/usr/bin ./bin/wake build default
Pass "BUILD"
WAKE_PATH=/usr/bin ./bin/wake install ../install
Pass "INSTALL"
> ls ../install/bin 
job-cache  wake  wakebox  wake-format
```

Is that what you mean, or do we also want to copy from `./bin/wake-format.native-cpp14-release` to `./bin/wake-format`

Reply:
That's all that's needed. I think you've satisfied the requirement.